config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml

```diff
@@ -0,0 +1,59 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: volumereplicationclasses.replication.storage.openshift.io
+spec:
+  group: replication.storage.openshift.io
+  names:
+    kind: VolumeReplicationClass
+    listKind: VolumeReplicationClassList
+    plural: volumereplicationclasses
+    singular: volumereplicationclass
+  scope: Namespaced
```

Comment:
Shouldn't `VolumeReplicationClass` be Cluster Scoped?

Reply:
Yes it should be cluster scoped

Reply:
I think @raghavendra-talur is addressing as part of [this](https://github.com/kube-storage/volume-replication-operator/pull/3/commits/5b20f86ffbd35c8ec7da8cb1e4584ccb3bdccd2e#diff-7044490c8cb898854279e3ce0ff4f06a016d17e3513ab044d9eafd05398700c3R52) change.